crates/node/ethstats/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+//! Support Ethstats
+
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth-docs.png",
+    html_favicon_url = "https://avatars0.githubusercontent.com/u/97369466?s=256",
+    issue_tracker_base_url = "https://github.com/paradigmxyz/reth/issues/"
+)]
+
+pub mod events;
+pub mod websockets;
```

Comment:
this we can rename to just connection maybe

if you need some reference for websocket, you can take a look at tokio-tungstenite examples

https://github.com/snapview/tokio-tungstenite/blob/master/examples/client.rs

and maybe even alloy:

https://github.com/alloy-rs/alloy/blob/c47d34bff1e23d1fc018cc1b652b3526f1254597/crates/transport-ws/src/native.rs#L72-L72

https://github.com/alloy-rs/alloy/blob/c47d34bff1e23d1fc018cc1b652b3526f1254597/crates/transport-ws/src/native.rs#L115-L116